drift/constants.py

```diff
@@ -10,3 +11,11 @@
 COMPARISON_SAME = "SAME"
 COMPARISON_DIFFERENT = "DIFFERENT"
 COMPARISON_INCOMPLETE_DATA = "INCOMPLETE_DATA"
+
+SYSTEM_PROFILE_INTEGERS = {'number_of_cpus', 'number_of_sockets',
+                           'cores_per_socket', 'system_memory_bytes'}
+SYSTEM_PROFILE_STRINGS = {'infrastructure_type', 'infrastructure_vendor',
+                          'bios_vendor', 'bios_version', 'bios_release_date',
+                          'os_release', 'os_kernel_version', 'arch', 'last_boot_time'}
+SYSTEM_PROFILE_LISTS_OF_STRINGS_ENABLED = {'cpu_flags', 'kernel_modules', 'enabled_services'}
+SYSTEM_PROFILE_LISTS_OF_STRINGS_INSTALLED = {'installed_services'}
```

Comment:
Why do we need to call out every field vs. just processing all fields presented by the system profile API?

Reply:
I split them out based on how they are parsed. For example, I have to cast `str()` on integers, and I have to do some additional parsing on `installed_services` so they show up as `service: installed` and not `service: true`

I could probably get rid of these sets for one-offs like `installed_services`. I didn't use a constant for `installed_packages` either.

Reply:
Thanks for the explanation! This sounds good